renderer/components/Onboarding/Steps/BackupSetup.js

```diff
@@ -7,14 +7,20 @@ import { Form, RadioGroup, Heading, Bar } from 'components/UI'
 import { BACKUP_FORM_WIDTH, BACKUP_FORM_HEIGHT } from './components/settings'
 import BaseBackupTypeItem from './components/BackupTypeItem'
 import Container from './components/Container'
+import SkipBackupsDialog from './components/SkipBackupsDialog'
 
 import messages from './messages'
 
 const BackupTypeItem = withFieldApi('backupType')(BaseBackupTypeItem)
 
 class BackupSetup extends React.Component {
   static propTypes = {
+    // eslint-disable-next-line react/boolean-prop-naming
+    canSkip: PropTypes.bool,
```

Comment:
This prop isn't used anywhere so should probably either be considered or removed.